CodeSniffer/Standards/Squiz/Sniffs/WhiteSpace/MemberVarSpacingSniff.php

```diff
@@ -102,22 +118,34 @@ protected function processMemberVar(PHP_CodeSniffer_File $phpcsFile, $stackPtr)
 
         $prev       = $phpcsFile->findPrevious(PHP_CodeSniffer_Tokens::$emptyTokens, ($first - 1), null, true);
         $foundLines = ($tokens[$first]['line'] - $tokens[$prev]['line'] - 1);
-        if ($foundLines === 1) {
+        $error      = 'Expected %s blank line before member var; %s found';
+        $expected   = $this->spacing;
+        $isFirst    = $tokens[$prev]['code'] === T_OPEN_CURLY_BRACKET;
+
+        // We allow specifying a different number of spaces to the first operator.
+        if ($isFirst === true) {
+            $expected = $this->firstMemberSpacing;
+            $error    = 'Expected %s blank lines before first member var; %s found';
+        }
+
+        if ($foundLines === $this->spacing || ($isFirst === true && $foundLines === $this->firstMemberSpacing)) {
```

Comment:
I found a bug with this line...

If I have a case where `$this->spacing` is set to `1` and `$this->firstMemberSpacing` is set to `0` and `$isFirst` is `true`, then if I happen to have exactly one space above the first member, I would expect to receive an error saying:

    Expected 0 blank lines before first member var; 1 found

However, since `$foundLines === $this->spacing` is evaluated first, and it is `true` (since `$foundLines` is `1`), then no error occurs.

I think the proper approach would be to change this line to:

``` php
if ($foundLines === $expected) {
```